src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -149,9 +149,11 @@ export class VisualizeEmbeddable
     }
 
     this.subscriptions.push(
-      this.getUpdated$().subscribe(() => {
+      this.getUpdated$().subscribe((value) => {
         const isDirty = this.handleChanges();
-        if (isDirty && this.handler) {
+        // we shouldn't try to update handler if we have error
+        // because it can lead to show error twice
+        if (isDirty && this.handler && !get(value, 'error.stack')) {
```

Comment:
No need to do any change here, as the popup is not displayed for runtime fields errors

Reply:
removed it